server/neptune/gateway/event/pull_request_review_handler.go

```diff
@@ -55,18 +58,19 @@ func (p *PullRequestReviewWorkerProxy) handle(ctx context.Context, event PullReq
 	}
 
 	// Ignore non-approval events
-	if event.Action != Approved {
+	if event.State != Approved {
+		p.Logger.InfoContext(ctx, fmt.Sprintf("not an approval event: %s", event.State))
 		return nil
 	}
 
 	// Ignore PRs without failing policy checks
 	failedPolicyCheckRuns, err := p.CheckRunFetcher.ListFailedPolicyCheckRuns(ctx, event.InstallationToken, event.Repo, event.Ref)
 	if err != nil {
-		p.Logger.ErrorContext(ctx, "unable to allocate policy v2")
+		p.Logger.ErrorContext(ctx, "unable to list failed policy check runs")
 		return err
 	}
 	if len(failedPolicyCheckRuns) == 0 {
-		p.Logger.ErrorContext(ctx, "no failed policies, skipping event")
+		p.Logger.InfoContext(ctx, "no failed policies, skipping event")
```

Comment:
same here as well, do we need an `Info` log is `failedPolicyCheckRuns` is empty?

Reply:
it's only for initial testing, will remove before rollout